### ISSUE TITLE
Where’s the latest docker release?

### ISSUE THREAD

**Issue by fenty17** [tailscale/tailscale#13474]
The latest version 1.74.0 isn’t yet available as docker container. I think I read somewhere docker releases are intentionally delayed a bit but the release was a couple of days ago so just checking if there’s an issue or other reason?

**Comment by agottardo** [comment]
You're correct that containers releases are intentionally delayed to ensure we do not introduce regressions. There are currently no blocking issues. We'll share more updates here when available.

**Comment by alexwlchan** [comment]
We did publish the 1.74 container release (over a year ago), and we continue to publish containers for new versions, so closing as fixed. If there are issues with newer container releases, we can file a new ticket.
